src/tests/tests/background.js

```diff
@@ -357,4 +358,53 @@ QUnit.test("subdomains on the yellowlist are preserved", (assert) => {
   );
 });
 
+QUnit.test("migrates misattributed MDFP domains from snitch map", (assert) => {
+  setupBadgerStorage(badger);
```

Comment:
We don't make use of any of the test fixture data in `setupBadgerStorage()` in this test ...